### PR TITLE
Fix setEnable method typo

### PR DESCRIPTION
**Pull request by mlykotom**
Getter and setter name don't share the same convention - _isEnable**d**_ vs _setEnable_,  therefore in kotlin, you can't call this as property.

Question is whether to keep `setEnable` method with deprecation for backwards compatibility.

### CODE REVIEW

File: constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/MotionScene.java [hunk 1]
```diff
@@ -648,7 +648,7 @@ public boolean isEnabled() {
          *
          * @param enable
          */
-        public void setEnable(boolean enable) {
+        public void setEnabled(boolean enable) {
```
Comment (kaeawc) [inline constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/MotionScene.java:651]: This is part of the existing public API in stable versions, definitely need to keep the existing method and add deprecation notice to make this change.

File: constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/ViewTransition.java [hunk 2]
```diff
@@ -502,7 +502,7 @@ boolean isEnabled() {
         return !mDisabled;
     }
 
-    void setEnable(boolean enable) {
+    void setEnabled(boolean enable) {
```
Comment (kaeawc) [inline constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/ViewTransition.java:505]: This is part of the public API, but only recently introduced in the `2.1.0-alphaX` work - @camaelon @jafu888  what do you think?

**Review by kaeawc — COMMENTED** [review]
I think it's worth deprecating the old methods. [Here is an example](https://github.com/androidx/constraintlayout/blob/2cdcac7394ae96855035321433021f62d249eee5/constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/widget/Barrier.java#L224).

File: constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/MotionScene.java (hunk 1 above)
Comment (mlykotom) [reply constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/MotionScene.java:665]: Ok, I put it back with deprecation.

File: constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/ViewTransition.java (hunk 2 above)
Comment (mlykotom) [reply constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/ViewTransition.java:505]: I don't think it's part of public API - method is package private.

Comment (jafu888) [reply constraintlayout/constraintlayout/src/main/java/androidx/constraintlayout/motion/widget/ViewTransition.java:505]: Change it

**Review by camaelon — APPROVED** [review]

**Pull request merged**